examples/image_processing/inplace/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM iron/ruby
```

Comment:
Maybe use [`iron/ruby:dev`](https://github.com/iron-io/dockers/blob/master/ruby/dev/Dockerfile) to avoid separate installs of `ruby-dev`, `bundler`, etc ?

Reply:
ah yes, that makes sense. Thank you !